data-pipeline/Gemfile

```diff
@@ -1,5 +1,7 @@
 source 'https://rubygems.org'
 
+ruby '2.5.3'
+
```

Comment:
Wasted a few hours trying to work out why the pipeline was broken - hey, lambda doesn't support ruby 2.6.3 so had to ensure that the pipeline stuff carries on with 2.5